internal/core/items/flickable.rs

```diff
@@ -41,11 +41,11 @@ use num_traits::Float;
 #[derive(FieldOffsets, Default, SlintElement)]
 #[pin]
 pub struct Flickable {
-    pub x: Property<LogicalLength>,
-    pub y: Property<LogicalLength>,
-    pub width: Property<LogicalLength>,
-    pub height: Property<LogicalLength>,
-    pub viewport: Empty,
+    pub viewport_x: Property<LogicalLength>,
+    pub viewport_y: Property<LogicalLength>,
+    pub viewport_width: Property<LogicalLength>,
+    pub viewport_height: Property<LogicalLength>,
```

Comment:
Nice to see that this now gets rid of the viewport hacks in the compiler.

Reply:
they were not "hacks" :-)